[FEATURE] CY_phishing_NOV ajout mention cybermalveillance

asilnomis opened the pull request:
## ❄️ Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## 🛷 Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## ☃️ Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🧑‍🎄 Pour tester
https://app-pr14438.review.pix.fr/modules/975d25d1/cyber-message-arnaque/details
<!-- Les instructions pour reproduire le problème, les profils de test, le parcours spécifique à utiliser, etc. -->

asilnomis committed:
[FEATURE] CY_phishing_NOV ajout mention cybermalveillance

pix-bot-github commented:
Choisir les applications à déployer :

- [x] [API](https://api-pr14438.review.pix.fr/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr14438/environment) <!-- pix-api-review -->
- [x] [App (.fr)](https://app-pr14438.review.pix.fr) – [App (.org)](https://app-pr14438.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-app-review-pr14438/environment) <!-- pix-app-review -->
- [ ] [Orga (.fr)](https://orga-pr14438.review.pix.fr) – [Orga (.org)](https://orga-pr14438.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-orga-review-pr14438/environment) <!-- pix-orga-review -->
- [ ] [Certif (.fr)](https://certif-pr14438.review.pix.fr) – [Certif (.org)](https://certif-pr14438.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-certif-review-pr14438/environment) <!-- pix-certif-review -->
- [ ] [Junior](https://junior-pr14438.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-junior-review-pr14438/environment) <!-- pix-junior-review -->
- [ ] [Admin](https://admin-pr14438.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-admin-review-pr14438/environment) <!-- pix-admin-review -->
- [ ] [API MaDDo](https://pix-api-maddo-review-pr14438.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr14438/environment) <!-- pix-api-maddo-review -->
- [ ] [Audit Logger](https://pix-audit-logger-review-pr14438.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr14438/environment) <!-- pix-audit-logger-review -->

> [!IMPORTANT]
> N'oubliez pas de déployer l'API pour pouvoir accéder aux fronts et/ou à l’API MaDDo.

asilnomis committed:
test italique

Updated the description field to include emphasis on the partnership with cybermalveillance.gouv.fr.

FranckRio committed:
Update CYPhishing_NOV.json

pix-service commented:
:tada: This PR is included in version 5.282.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.282.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
